docs/rofl/prerequisites.md

```diff
@@ -77,13 +77,28 @@ nightly-2022-08-22-x86_64-unknown-linux-gnu (overridden by '/code/rust-toolchain
 rustc 1.65.0-nightly (c0941dfb5 2022-08-21)
 ```
 
+ROFL binaries should be compiled for [MUSL C standard library]. You will need
```

Comment:
I'm not sure why they should be compiled for musl, because this can actually cause some problems. We could mention that they need to be compiled for musl in case of Localnet, but outside that it doesn't make sense to require musl?

Reply:
For Testnet and Mainnet production builds the build target is `x86_64-fortanix-unknown-sgx` right? I don't see other environments developers would want to run their ROFLs?

Reply:
Agree, I'll change the wording ;)

Reply:
Yeah so the issue is that for Mainnet and Testnet the ELF binary doesn't really matter and requiring MUSL for those doesn't make sense. The orcs currently must include the ELF binaries as well -- this will change in Oasis Core 24.3 where ELF binaries will become optional and not having them will automatically force the use of SGX/TDX.